README.md

```diff
@@ -48,3 +49,30 @@ Here are two simple examples of the -a operand:
 
        gogrep -x '$x + $y'                   // will match both numerical and string "+" operations
        gogrep -x '$x + $y' -a 'type(string)' // matches only string concatenations
+
+The attributes understood by the -a operand are as follows:
+
+	comp
+		The node's type is comparable
+	addr
+		The node's type is addressable
+	rx(regexp)
+		The node is an identifier that matches the regular expression.
+	type(type)
+		The node has the given Go type.
+		TODO how are packages referred to?
```

Comment:
packages are referred to as per the scope where the node is located, if I recall correctly.

I also seem to recall this is an area that I wasn't too happy with. The ideal solution might be local scope plus some sort of fallback, like goimports-like resolution. Or perhaps something more explicit, like what rsc/rf does.